### PR TITLE
generic install script

### PR DESCRIPTION
**Pull request by robkooper**

### CODE REVIEW

File: scripts/install_pecan.sh [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+echo "######################################################################"
+echo "POSTGRES"
+echo "######################################################################"
+#    ADD export PATH=${PATH}:/usr/pgsql-9.5/bin
+#    ADD exclude=postgresql* to /etc/yum.repos.d/CentOS-Base.repo or /etc/yum/pluginconf.d/rhnplugin.conf 
+#    SEE https://wiki.postgresql.org/wiki/YUM_Installation#Configure_your_YUM_repository
+case "$OS_VERSION" in
+  RH_5)
+    echo "No PostgreSQL configuration (yet) for RedHat 5"
+    exit 1
```
Comment (dlebauer) [inline scripts/install_pecan.sh]: how about adding 'but the database can be run on a separate server from PEcAn; (see reference)'

Then either exit or 'continuing to install PEcAn without database' 

That said, I'm not sure how important it is to support RH 5

**Comment by dlebauer** [comment]
This looks great and could save lots of people lots of time since installing can be such a hassle. Have you tested it on all of these systems?


A few thoughts (not suggestions for the pull request)

* why this script instead of Docker (seems Docker would be easier).
* would be great to have support for OSX and Windows Linux Subsystem (!)
* could separate the BETYdb and PEcAn installations, where the PEcAn install could be given parameters for the BETYdb instance being used.

**Comment by robkooper** [comment]
this is my script to make VM's, I created (once) a RH-6 and RH-7 machine, but this is more for people to look at how I do things, this is not a generic install PEcAn script.

**Comment by mdietze** [comment]
But it is a good starting point for a more generic install script 👍

**Pull request merged**